streaming/src/main/scala/org/apache/spark/streaming/ui/StreamingPage.scala

```diff
@@ -392,8 +392,12 @@ private[ui] class StreamingPage(parent: StreamingTab)
       maxX: Long,
       minY: Double,
       maxY: Double): Seq[Node] = {
+    val maxYCalculated = listener.receivedEventRateWithBatchTime.map {
+      case (streamId, eventRates) => new EventRateUIData(eventRates).max.getOrElse(0.0)
```

Comment:
actually you can probably just do
```
val maxYCalculated = listener.receivedEventRateWithBatchTime.values
  .map { case (_, eventRate) => eventRate }
  .reduceOption(math.max)
  .map(_.ceil.toLong)
  .getOrElse(0L)
```
is it the same? Then we don't have to create a new `EventRateUIData` every time. It's also more readable IMO.